rust/xaynet-analytics/src/database/analytics_event/repo.rs

```diff
@@ -44,11 +40,11 @@ impl<'screen, 'db> Repo<AnalyticsEvent<'screen>> for AnalyticsEventRepo<'db> {
 pub struct MockAnalyticsEventRepo {}
 
 impl<'screen> Repo<AnalyticsEvent<'screen>> for MockAnalyticsEventRepo {
```

Comment:
Makes sense. Actually right now they're not currently used, cause `DataCombiner` doesn't need them any more. But i left them there just in case. Is it OK if i clean them up in the upcoming unit tests PR? I will then either removed if not needed, or move them under the relevant tests mod, or I think Robert mentioned there's a library might help with this.

Reply:
a later cleanup is completely ok. the crate robert mentioned is [mockall](https://crates.io/crates/mockall), it's pretty powerful, he used it also in the xaynet state machine tests refactoring, see #708.